InformationScripting/src/query_framework/QueryResultVisualizer.cpp

```diff
@@ -121,6 +121,22 @@ Optional<int> QueryResultVisualizer::visualize(const TupleSet& ts)
 		}
 	}
 
+	for (auto changeTuple : ts.tuples("change"))
```

Comment:
This shouldn't really be here, the visualizer should be more generic.

That also applies to the `calls` handling above.